doc/user/bgp.rst

```diff
@@ -2895,6 +2895,89 @@ Displaying Update Group Information
 
    Display Information about update-group events in FRR.
 
+Displaying Information for all address families
+-----------------------------------------------
+The following commands allow displaying routes or configured information for all
+address families or for the specified AFI.
+This is useful when we do not want to wade individually through the AFI/SAFs.
+To achieve this ``all`` option is added to the following show bgp commands.
+
+.. index:: show [ip] bgp [ipv4|ipv6] [all] [wide|json]
```

Comment:
Could you extend existing documentation with relevant options? http://docs.frrouting.org/en/latest/bgp.html#clicmd-showipbgp[wide]

Reply:
Sorry. Do you mean, instead of writing the commands separately with all option here, 
you want me to update the existing commands itself ?

Reply:
right :)

Reply:
ok will do that. thanks :)

Reply:
addressed this comment